dotnet-grad/Models/UserConfiguration.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace dotnet_grad.Models
+{
+  public class UserConfiguration
+  {
+    public void Configure(EntityTypeBuilder<UserModel> builder)
+    {
+      builder.OwnsMany(x => x.organsiation_models);
```

Comment:
sies. According to the AC, organisations can exist without users, using `OwnsMany` means the exact opposite. The main difference between a vanilla relationship IE `HasOne()/HasMany()` and `OwnsOne()/OwnsMany()` is that owned types, can not exist without their owners ( in this case the user is the owner of the organisation), other that that it just gets auto included ( joined) and will always be present when querying a user and need not be explicitly done.